mkdocs.yml

```diff
@@ -105,4 +106,4 @@ edit_uri: edit/main/docs/
 site_url: https://microsoft.github.io/CSS-Exchange
 plugins:
   - search
-  - git-revision-date-localized
+  - git-revision-date-localized
```

Comment:
Please add the newline back to the end of the file (the diff shows the red circle with the line when the end-of-file newline is missing).

Reply:
done